### PR TITLE
Updates for onboarding instructions

### PR DESCRIPTION
**Pull request by kinghorn**
Several updates in content and formatting to the onboarding document. Minor update to the Limitations document.

### CODE REVIEW

File: prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md [hunk 1]
```diff
@@ -1,79 +1,115 @@
-# Instructions for Private Preview onboarding
+# Instructions for Azure Managed Service for Prometheus Private Preview onboarding
 
-**1. The steps need to executed in this order. If not, step 1 has to be rerun after executing step 2**
+**1. The steps need to be executed in the order shown below.**
```
Comment (rashmichandrashekar) [inline prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md:3]: Scott, we might probably want to keep the statement - "If not, step 1 has to be rerun after executing step 2", just to make it obvious?

File: prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md [hunk 2]
```diff
@@ -1,79 +1,115 @@
...
 
-        wget https://raw.githubusercontent.com/microsoft/Docker-Provider/prometheus-collector/prometheus-collector/MAC-3P-Docs/RoleDefinition.json
+**9. Azure Managed Service for Prometheus if free to use for ingestion of data and querying of data  while in Private Preview**
+
+<br/>
+<br/>
+
+## **Step 1**: Create your Azure Prometheus and Azure Grafana resources.
+
+This step creates the Monitoring account used to store your Promethems metrics. It also creates your Azure Managed Grafana workspace. These resources are regional, therefore this step needs to be run once for each region where you want to create these resources (East US, East US2, West Europe). Onboarding to more than one region is optional.
```
Comment (rashmichandrashekar) [inline prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md:26]: type: prometheus

Comment (kinghorn) [reply prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md]: fixed typo

File: prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md (hunk 1 above)
Comment (kinghorn) [reply prometheus-collector/MAC-3P-Docs/Instructions for Private Preview Onboarding.md]: added info back in about re-running step 1

**Review by rashmichandrashekar — APPROVED** [review]

**Pull request merged**